srcpkgs/hlint/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'hlint'
 pkgname=hlint
-version=2.1.11
-revision=3
+version=2.2.4
+revision=1
 build_style=haskell-stack
-stackage="lts-13.2"
+stackage="nightly-2019-11-02"
```

Comment:
We stick to latest lts, because we prefer stable software.

Reply:
K having to close this, as the needed deps don't exist in the latest LTS.